crates/compiler/can/tests/snapshots/test_suffixed__suffixed_tests__bang_in_pipe_root.snap

```diff
@@ -4,16 +4,16 @@ expression: snapshot
 ---
 Defs {
     tags: [
-        Index(2147483648),
+        EitherIndex(2147483648),
     ],
     regions: [
         @0-28,
     ],
     space_before: [
-        Slice(start = 0, length = 0),
+        Slice<roc_parse::ast::CommentOrNewline> { start: 0, length: 0 },
     ],
     space_after: [
-        Slice(start = 0, length = 1),
+        Slice<roc_parse::ast::CommentOrNewline> { start: 0, length: 1 },
```

Comment:
Not directly related to this PR, but I see a couple problems with the snapshot in this way:

1. Marking indexes/slices doesn't tell you much other than how the implementation works. I don't think this is useful for snapshot tests. It's probably better to print a debug representation of the AST that is materialized, or just print back the program fully and make sure it looks reasonable. (If there's ever a source of non-determinism in the parsing, e.g. parallel parsing, the indexes are also going to be non deterministic)
2. Embedding the type names is going to create a big diff every time there's a refactor, like is done here. Probably worth avoiding that.

Reply:
Yeah that's fair - I took the type name back out 👍